assembly/std/arguments.ts

```diff
@@ -0,0 +1,165 @@
+import {Address} from './address';
+import {ByteArray} from '@massalabs/as/assembly/byteArray';
+
+/**
+ * Args for remote function call
```

Comment:
You should explain in few words what this class is doing.

Reply:
done